pipelines/deduplication/acteur_deduplicate.py

```diff
@@ -0,0 +1,223 @@
+# PARAMETERS
+# ------------------------------------------
+SOURCES_PREFERRED_CODES = ["ALIAPUR", "COREPILE"]
+ENVIRONMENTS = ["DEV", "PREPROD", "PROD"]
+# URLs de la DB
+DB_URLS = {
+    "DEV": os.environ.get("DB_URL_DEV"),
+    "PREPROD": os.environ.get("DB_URL_PREPROD"),
+    "PROD": os.environ.get("DB_URL_PROD"),
+}
+# Chemin vers le fichier CSV
+CURRENT_DIR = Path(__file__).resolve().parent
+CLUSTERING_CSV_FILEPATH = CURRENT_DIR / "./20241121_dedup_dechetteries.csv"
+
+# Colonnes pour le debug
+COLS_DEBUG = ["cluster_id", "identifiant_unique", "children_count", "source_id"]
+
+
+def environment_setup() -> Tuple[Engine, list, Path]:
+    """Choix de l'environnement et retourne les paramètres nécessaires
+
+    Returns:
+        db_engine (Engine): connexion à la base de données
+        clusters_done_list: liste des clusters traités
+        clusters_done_path: chemin vers le fichier de suivi des clusters traités
+    """
+
+    banner("Setup de l'environnement")
+    print("DB_URLS:")
+    for env, url in DB_URLS.items():
+        print(f"{env}={url}")
+    assert any(
+        DB_URLS.values()
+    ), f"""URLs DB manquantes: besoin os.environ sur 1 de
+        {["DB_URL_" + env for env in ENVIRONMENTS]}"""
+
+    env_name = Prompt.ask("Environnement", choices=ENVIRONMENTS)
+    db_url = DB_URLS[env_name]
+    assert db_url, "db_url est vide"
+    db_engine = create_engine(db_url)
+
+    # Fichier de suivi des clusters traités construit à partir de l'environnement
+    clusters_done_path = CURRENT_DIR / f"./clusters_done_{env_name.lower()}.json"
+    clusters_done_list = []
+    if not clusters_done_path.exists():
+        clusters_done_path.write_text(json.dumps(clusters_done_list, indent=4))
+    else:
+        clusters_done_list = json.loads(clusters_done_path.read_text())
+
+    print(f"{env_name=}")
+    print(f"{db_url=}")
+    print(f"{db_engine=}")
+    print(f"{clusters_done_path=}")
+    print(f"{len(clusters_done_list)=}")
+
+    # Doesn't understand that db_engine is of proper type Engine
+    return db_engine, clusters_done_list, clusters_done_path  # type: ignore
+
+
+def main() -> None:
+    """Fonction principale qui regroupe toute là logique de déduplication"""
+    banner(__file__)
+
+    DRY_RUN = Confirm.ask("DRY RUN (càd pas de modifications DB)?")
+
+    print(f"{DRY_RUN=}")
+    if not DRY_RUN:
+        if not Confirm.ask("\n🔴 Modifications DB à venir: continuer? 🔴"):
+            raise SystemExit("Interrompu par l'utilisateur")
+
+    # ------------------------------------------
+    # SETUP D'ENVIRONNEMENT
+    # ------------------------------------------
+    db_engine, clusters_done_list, clusters_done_path = environment_setup()
+
+    # ------------------------------------------
+    # DONNEES D'ENTREE
+    # ------------------------------------------
+    # Chaque ligne est un cluster_id <-> acteur identifiant_unique
+    df_src = source_data_get(CLUSTERING_CSV_FILEPATH)
```

Comment:
```suggestion
    df_source_data = keep_only_cluster_id_and_identifiant_unique_from(CLUSTERING_CSV_FILEPATH)
```

Idem ici, je pense qu'on peut s'affranchir de commentaires si les signatures des fonctions reflètent bien ce qu'on fait, par exemple 

C'est verbeux, un peu _stupide_ dans l'approche, mais 
- `df_source_data` : un data frame avec les données d'entrée
- `keep_only_cluster_id_and_identifiant_unique_from` : on sait ce que ça fait 
le code s'auto documente, pas de commentaire à maintenir, les docstring peuvent être maintenues à une version très synthétique